doc/nrf/known_issues.rst

```diff
@@ -2739,6 +2739,16 @@ NCSDK-14015: Execution halts during boot
 NCSDK-13949: TF-M Secure Image copies FICR to RAM on nRF9160
   TF-M Secure Image copies the FICR to RAM address between ``0x2003E000`` and ``0x2003F000`` during boot on nRF9160.
 
+Samples
+*******
+
+.. rst-class:: v2-2-0
+
+NCSDK-18847: :ref:`radio_test` sample does not build with support for Skyworks front-end module.
+  When bulding sample with support for different front-end module than the nRF21540, the sample uses not existing configuration to initialize Tx power data which cause compilation error also source file with code for generic front-end module is not included into build.
```

Comment:
```suggestion
  When building the sample with support for different front-end module than nRF21540, it uses a non-existing configuration to initialize Tx power data.
  This causes a compilation error, because the source file with code for a generic front-end module is not included in the build.
```

Reply:
Done